addons/skin.estuary/xml/Includes_DialogSelect.xml

```diff
@@ -624,15 +624,15 @@
 			<control type="list" id="6">
 				<left>20</left>
 				<top>100</top>
-				<width>880</width>
+				<width>930</width>
```

Comment:
Same as above.

Reply:
Changed that should have been reverted, fixed.